code/go/0chain.net/core/util/columnizer.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"strings"
+)
+
+var (
+	firstCapRegex 	= regexp.MustCompile("(.)([A-Z][a-z]+)")
+	allCapRegex 	= regexp.MustCompile("([a-z0-9])([A-Z])")
+	gormColumRegex 	= regexp.MustCompile("(?m)column:([a-zA-Z0-9_]+);")
+)
+
+
+func toSnakeCase(str string) string {
+    snake := firstCapRegex.ReplaceAllString(str, "${1}_${2}")
+    snake = allCapRegex.ReplaceAllString(snake, "${1}_${2}")
+    return strings.ToLower(snake)
+}
+
+func Columnize[T any](objects []T) (map[string][]interface{}, error) {
```

Comment:
We shall avoid putting code in `util` package when possible, we can put it in `dbs/event`, where the `updater.go` located. Or have its own package in `dbs`.